StRoot/StEvent/StFttCluster.h

```diff
@@ -48,17 +61,23 @@ class StFttCluster : public StObject {
     const StPtrVecFttPoint& points() const;
     void addPoint(StFttPoint* p);
     // void print(Option_t *option="") const;
+    void print();
 
 private:
-    Int_t mId=-1;             // Eventwise cluster ID
+    Int_t   mId=-1;             // Eventwise cluster ID
     UChar_t mPlane;
     UChar_t mQuadrant;
     UChar_t mRow;
     UChar_t mOrientation = kFttUnknownOrientation;        // Orientation of cluster
-    Int_t mNStrips=0;         // Number of strips
+    Int_t   mNStrips=0;         // Number of strips
     Float_t mSumAdc=0.0;      // Total ADC (0th moment)
-    Float_t mX=0.0;             // Mean x ("center of gravity") in local grid coordinate (1st moment)
-    Float_t mSigma=0.0;        // 2nd moment
+    Float_t mX=-999;             // Mean x ("center of gravity") in local grid coordinate (1st moment)
+    Float_t mSigma=-999;        // 2nd moment
+    Float_t mMaxADC = -999;     // ADC of the strip with maximum ADC
+    Int_t   mIndexMaxStrip = -999;// index of the strip with maximum ADC
+    Float_t mMaxStripCenter = -999;// Strip center of strip with maximum ADC
+    Float_t mMaxStripLeftEdge = -999;//left edge of the strip with maximum ADC
+    Float_t mMaxStripRightEdge = -999;//right edge of the strip with maximum ADC
```

Comment:
Line 85, increase ClassDef version to 3.

Reply:
See comment below, all changes related to FTT reverted since they should be handled in PR #569 .
Done with https://github.com/star-bnl/star-sw/pull/571/commits/6088f974a30af0141f460b3fdd3183ae4f7c66ec